src/coreclr/jit/flowgraph.cpp

```diff
@@ -2596,8 +2596,13 @@ PhaseStatus Compiler::fgAddSwiftErrorReturns()
 
         fgWalkResult PreOrderVisit(GenTree** use, GenTree* user)
         {
-            if ((*use)->OperIs(GT_LCL_VAR) && ((*use)->AsLclVarCommon()->GetLclNum() == m_compiler->lvaSwiftErrorArg))
+            if ((*use)->AsLclVarCommon()->GetLclNum() == m_compiler->lvaSwiftErrorArg)
             {
+                if (!(*use)->OperIs(GT_LCL_VAR))
+                {
+                    BADCODE("Expected GT_LCL_VAR of SwiftError* parameter, got other GenTreeLclVarCommon node");
```

Comment:
I think the message should be phrased in terms of IL, not IR constructs

Reply:
Would something like "Found invalid use of SwiftError* parameter" be too vague?

Reply:
That sounds good to me